framework/widgets/ActiveField.php

```diff
@@ -721,7 +721,7 @@ protected function getClientOptions()
         $options = [];
 
         $inputID = $this->getInputId();
-        $options['id'] = $inputID;
+        $options['id'] = Html::getInputId($this->model, $this->attribute);
```

Comment:
@cebe How exactly? As far as I can tell the issue was in css selectors which were fixed earlier but the `id` was fine and after the fix it's not. Any way this one fixes AJAX validation while client validation remains working. I checked it manually but I'll make some automatic tests this week to clear things out.